client/my-sites/media-library/filter-bar.jsx

```diff
@@ -63,6 +66,8 @@ export class MediaLibraryFilterBar extends Component {
 		const { translate } = this.props;
 
 		switch ( filter ) {
+			case 'this-post':
+				return translate( 'This Post', { comment: 'Filter label for media list' } );
```

Comment:
On the Pages list, this & other places that mention `Post` should say `Page` instead